client/src/Substack.tsx

```diff
@@ -30,26 +29,38 @@ const Substack: React.FC = () => {
       const html = document.createElement('html')
       const initialStyles = Array.from(document.querySelectorAll('style'))
       html.style.visibility = 'hidden'
-      html.style.overflow = 'hidden'
       html.innerHTML = replaceSubstackLink(page, { substackHost: pageId, subdomain, sld })
       // console.log(html.innerHTML)
       document.replaceChild(html, document.documentElement)
       const scripts = Array.from(document.querySelectorAll('script'))
       let loaded = 0
       const targetNumLoaded = scripts.filter(e => e.src?.includes('substack')).length
-      // const targetNumLoaded = scripts.length
+
       for (const script of scripts) {
         const newScript = document.createElement('script')
+
         if (script.src) {
           newScript.src = script.src
         }
+
         if (script.type) {
           newScript.type = script.type
         }
+
         if (script.innerHTML) {
           newScript.innerHTML = script.innerHTML
         }
+
+        if (script.noModule) {
+          newScript.noModule = true
+        }
```

Comment:
Need to keep `noModule` attribute for this script not working
![image](https://github.com/polymorpher/dot-country-embedder/assets/64591585/b18220fb-dbc8-4833-92aa-435c8d9041df)